scripts/sanitycheck

```diff
@@ -834,6 +834,9 @@ def main():
 
     suite = TestSuite(options.board_root, options.testcase_root, options.outdir)
 
+    # Check version of zephyr repo
+    suite.check_zephyr_version(ZEPHYR_BASE)
```

Comment:
Just a thought...ZEPHYR_BASE can be used as is because it it already set in the script. Need not be an argument to this function check_zephyr_version.

Reply:
Thanks. I modified the script accordingly.